packages/i18next/src/lib/functions.ts

```diff
@@ -198,17 +273,77 @@ export async function editLocalized<TKeys extends TFunctionKeys = string, TInter
 	target: Message,
 	options: LocalizedMessageOptions<TKeys, TInterpolationMap>
 ): Promise<Message>;
+/**
+ * Edits a deferred/replied interaction using the language `keys` from your i18next language setup.
```

Comment:
This is somewhat unclear. Are you trying to say "an interaction that has been replied to"? If you clarify what this is supposed to mean, I can help you find the best way to express it.

Reply:
Hmm yes since deferred is also means as replied

Reply:
To defer means to postpone or cede via an acknowledgment of superiority.
For instance, I can defer a meeting, or I can defer to your programming knowledge, but you wouldn't talk about someone deferring a message (because that would mean delaying the message).

Reply:
In any case, assuming this follows what you mean, here's a good rephrasal.

```suggestion
 * Edits an interaction or reply to an interaction using the given language `keys`.
```

Reply:
Hmm I means this is the [`BaseCommandInteraction#deferReply`](https://discord.js.org/#/docs/discord.js/stable/class/BaseCommandInteraction?scrollTo=deferReply) method

Reply:
Oh nope its this one [`BaseCommandInteraction#deferred`](https://discord.js.org/#/docs/discord.js/stable/class/BaseCommandInteraction?scrollTo=deferred)

Reply:
My apologies. In that case, you appear to have misinterpreted Discord's use of the word "deferred". It is in fact a delay since they postpone the reply by showing a waiting message.

Reply:
Ic thanks

Reply:
My latest suggestion edit should capture the intended meaning correctly.